pkg/util/restoptions/configgetter.go

```diff
@@ -26,6 +26,10 @@ import (
 // UseConfiguredCacheSize indicates that the configured cache size should be used
 const UseConfiguredCacheSize = -1
 
+// DefaultCacheSize is the cache size to use for etcdHelper
+// TODO: Make this configurable in future
+const DefaultCacheSize = 100
```

Comment:
isn't this the same thing as `genericapiserver.DefaultDeserializationCacheSize`? Why not use that? Also, that defaults to 50000, which seems too big, but is 100 way too small?